suzieq/engines/pandas/engineobj.py

```diff
@@ -549,8 +557,9 @@ def _add_field_to_summary(self, field: str, method: str = 'nunique',
         if not field_name:
             field_name = field
         field_per_ns = getattr(self.nsgrp[field], method)()
-        {self.ns[i].update({field_name: field_per_ns.get(i, 0)})
-         for i in self.ns.keys()}
+        # pylint disable=expression-not-assigned
+        _ = {self.ns[i].update({field_name: field_per_ns.get(i, 0)})
+             for i in self.ns.keys()}
```

Comment:
Since you added the `_ =`, I don't think the `pylint disable` is needed

Reply:
Ack